webplugin/css/app/mck-sidebox-1.0.css

```diff
@@ -5966,3 +5980,19 @@ div#mck-rated:before{
 		opacity: 1;
 	}
 }
+.mck-mic-btn {
+	display: inline-block;
+	float: left;
+	margin-top: 12px;
+}
+.mck-mike-btn {
```

Comment:
@kumarabhinav002 what's the difference between mic and mike?

Reply:
Earlier we were using mck-mic-btn, will change to mck-mike-btn.

Reply:
@kumarabhinav002 if possible please change it to `mck-mic-button` and not `mike`.